pkg/vpa/service.go

```diff
@@ -149,6 +149,44 @@ func (c *Service) CreateTortoiseUpdaterVPA(ctx context.Context, tortoise *autosc
 	return vpa, tortoise, nil
 }
 
+func (c *Service) UpdateVPACRP(ctx context.Context, tortoise *autoscalingv1beta3.Tortoise) (*v1.VerticalPodAutoscaler, error) {
```

Comment:
nit: CRP is not a common name. It's more readable to name it `UpdateVPAContainerResourcePolicy` or something.